pertpy/tools/_distances/_distances.py

```diff
@@ -856,3 +914,97 @@ def onesided_distances(
 
     def from_precomputed(self, P: np.ndarray, idx: np.ndarray, **kwargs) -> float:
         raise NotImplementedError("ClassifierClassProjection cannot be called on a pairwise distance matrix.")
+
+
+class MeanVarDistnDistance(AbstractDistance):
+    """
+    Distance betweenv mean-var distributions of gene expression.
+
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.accepts_precomputed = False
+
+    def __call__(self, X: np.ndarray, Y: np.ndarray, **kwargs) -> float:
+        """
+        Difference of mean-var distibutions in 2 matrices
+        Args:
+            X: Normalized and log transcformed cells*genes count matrix.
+            Y: Normalized and log transcformed cells*genes count matrix.
+        """
+
+        # Get log mean & var for comparison
+        def _mean_var(x, log: bool = False):
+            mean = np.mean(x, axis=0)
+            var = np.var(x, axis=0)
+            positive = mean > 0
+            mean = mean[positive]
+            var = var[positive]
+            if log:
+                mean = np.log(mean)
+                var = np.log(var)
+            return mean, var
+
+        def _prep_kde_data(x, y):
+            return np.concatenate([x.reshape(-1, 1), y.reshape(-1, 1)], axis=1)
+
+        def _grid_points(d, n_points=100):
+            # Make grid, add 1 bin on lower/upper end to get final n_points
+            d_min = d.min()
+            d_max = d.max()
+            # Compute bin size
+            d_bin = (d_max - d_min) / (n_points - 2)
+            d_min = d_min - d_bin
+            d_max = d_max + d_bin
+            return np.arange(start=d_min + 0.5 * d_bin, stop=d_max, step=d_bin)
+
+        def _parrallel_score_samples(kde, samples, thread_count=int(0.875 * multiprocessing.cpu_count())):
```

Comment:
```suggestion
        def _parallel_score_samples(kde, samples, thread_count=int(0.875 * multiprocessing.cpu_count())):
```

Reply:
maybe a small comment on why the 0.875 for future maintenance?
For example

"the thread_count is determined using the factor 0.875 as recommended here [insert link]"
or
"the thread_count is determined as it seems to work well here [insert link]"